Lib/calendar.py

```diff
@@ -765,6 +791,7 @@ def main(args=None):
         sys.exit(1)
 
     locale = options.locale, options.encoding
+    today = datetime.date.today()
```

Comment:
We might want a CLI arg for disabling colors instead of just relying on the environment variables.

Reply:
Yes, for example https://no-color.org/ says command-line arguments should override env vars (FAQ number 2). However, it's out of scope for this PR and would be its own feature.

Reply:
Seems we do not completely follow the `no-color.org`/`NO_COLOR` standard:

> Command-line software which adds ANSI color to its output by default should check for a `NO_COLOR` environment variable that, when present and not an empty string (regardless of its value), prevents the addition of ANSI color.

Setting `NO_COLOR` to the empty string prevents colour (both in the REPL and for this PR).

Reply:
Ah right, this was changed in the `NO_COLOR` spec a couple of years ago:

https://github.com/jcs/no_color/commit/99f90e27d0fac6a34d07998f8f577e1f2f620c61

We could update `_colorize.py` to respect this separately as a bug fix.